CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules/")
 
+if(${CMAKE_BUILD_TYPE} MATCHES "Debug")
+  add_definitions(-DDEBUG)
```

Comment:
i don't understand where/how is `DEBUG` respected in ceph?  fwiw, we remove `-DNDEBUG` in `src/CMakeLists.txt`.